rust/ddlog_benches/Makefile.toml

```diff
@@ -1,6 +1,6 @@
 # Runs all ddlog benchmarks
 [tasks.benchmarks]
-dependencies = ["download-data", "bench-twitter", "bench-livejournal"]
+dependencies = ["bench-twitter", "bench-livejournal"]
```

Comment:
```suggestion
dependencies = [
    "download-twitter",
    "bench-twitter",
    "download-livejournal",
    "bench-livejournal",
]
```

Reply:
Is this really necessary? It looks to me like when you run 'cargo make benchmarks' these will be done through transitive dependencies anyway. For me it works fine as it is in this PR.